rebar.config

```
@@ -16,7 +16,9 @@
 %% OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 {deps,
-  [ {jsx, "v2.9.0"} ]}.
+  [ {jsx, "v2.9.0"}
+  , {iso8601, {git, "https://github.com/erlsci/iso8601.git", {tag, "1.3.3"}}}
```

we could get it from `hex`, just like `jsx`
```suggestion
  , {iso8601, "1.3.3"}
```